insights/parsers/qpid_stat.py

```diff
@@ -193,3 +195,123 @@ def search(self, **kwargs):
 
         """
         return keyword_search(self.data, **kwargs)
+
+
+@parser(Specs.qpid_stat_g)
+class QpidStatG(CommandParser):
+
+    """
+    This parser reads the output of the command ``qpid-stat -g
+    --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b
+    amqps://localhost:5671``
+
+    Sample output::
+
+        Broker Summary:
+          uptime           cluster       connections  sessions  exchanges  queues
+          =========================================================================
+          96d 23h 50m 41s  <standalone>  23           37        14         33
+
+        Aggregate Broker Statistics:
+          Statistic                   Messages    Bytes
+          ========================================================
+          queue-depth                 0           0
+          total-enqueues              1,726,798   42,589,932,236
+          total-dequeues              1,726,798   42,589,932,236
+          persistent-enqueues         28,725      23,889,836
+          persistent-dequeues         28,725      23,889,836
+          transactional-enqueues      0           0
+          transactional-dequeues      0           0
+          flow-to-disk-depth          0           0
+          flow-to-disk-enqueues       0           0
+          flow-to-disk-dequeues       0           0
+          acquires                    1,726,798
+          releases                    0
+          discards-no-route           41,163,896
+
+    Attributes:
+        data (list of dict): A list of dictionaries with the key-value data
+            from the table.
+        by_queue (dict of dict): A dictionary of the same data dictionaries
+            stored by cluster name or queue name.
+
+    Examples:
+        >>> type(qpid_stat_g)
+        <class 'insights.parsers.qpid_stat.QpidStatG'>
+        >>> type(qpid_stat_g.data) == type([])
+        True
+        >>> type(qpid_stat_g.data[0]) == type({}) # Each row is a dictionary formed from the table
+        True
+        >>> qpid_stat_g.data[0]['uptime']
+        '97d 0h 16m 24s'
+        >>> qpid_stat_g.data[0]['cluster']
+        '<standalone>'
+        >>> qpid_stat_g.data[1]['Statistic']
+        'queue-depth'
+        >>> qpid_stat_g.data[1]['Messages']
+        '0'
+        >>> qpid_stat_g.data[11]['Bytes']
+        ''
+        >>> type(qpid_stat_g.by_queue) == type({}) # Dictionary lookup by queue ID
+        True
+        >>> qpid_stat_g.by_queue['queue-depth'] == qpid_stat_g.data[1]
+        True
+        >>> enqueues = qpid_stat_g.search(Statistic__contains='enqueues')  # Keyword search
+        >>> type(enqueues) == type([])
+        True
+        >>> len(enqueues)
+        4
+        >>> enqueues[0] == qpid_stat_g.data[2]  # List contains matching items
+        True
+    """
+
+    def __iter__(self):
+        return iter(self.data)
+
+    def parse_content(self, content):
+        index = 0
```

Comment:
A much simple way to get the index could be as below, And I think an `Exception` should be raised when no such line is found.
```
index = [i for i, l in enumerate(content) if l.startswith("Aggregate Broker Statistics")]
if not index:
   raise ParseException("Incorrect Content..BLABLA....")
index = index[0]
```